examples/nlp/language_modeling/conf/megatron_retro_config.yaml

```diff
@@ -1,127 +1,260 @@
 defaults:
-  - .@model: megatron_model_base_config
+  - _self_
+  - optional tp_overlap@model.ub_tp_comm_overlap_cfg:
 
-name: test_retro
+name: megatron_gpt
 restore_from_path: null # used when starting from a .nemo file
 
 trainer:
-  devices: 2
+  devices: 1
   num_nodes: 1
   accelerator: gpu
   precision: 16
   logger: False # logger provided by exp_manager
   enable_checkpointing: False
   use_distributed_sampler: False
-  max_epochs: -1 # PTL default. In practice we don't usually train for more than 1 epoch.
+  max_epochs: -1 # PTL default. In practice, max_steps will be reached first. 
   max_steps: 100000 # consumed_samples = global_step * micro_batch_size * data_parallel_size * accumulate_grad_batches
   log_every_n_steps: 10
   val_check_interval: 100
-  limit_val_batches: null
-  limit_test_batches: null
-  accumulate_grad_batches: 1
+  limit_val_batches: 50
+  limit_test_batches: 500
+  accumulate_grad_batches: 1 # do not modify, grad acc is automatic for training megatron models
   gradient_clip_val: 1.0
+  benchmark: False
+  enable_model_summary: False # default PTL callback for this does not support model parallelism, instead we log manually
 
 exp_manager:
   explicit_log_dir: null
   exp_dir: null
-  name: megatron_retro
+  name: megatron_gpt
```

Comment:
you can get the name from the name filed at the top of the config using hydra interpolation

Reply:
Fixed.